City-of-Helsinki/drupal-helfi-platform-config
# UHF-5341: Remove TPR Units not located in Helsinki

How to test:

- set up any instance
- get this branch of the module: `composer require drupal/helfi_platform_config:dev-UHF-5341_remove_non-helsinki_tpr_units`
- import all TPR Units: `drush mim tpr_unit`
- see a large number of TPR Units in the admin view
- run the drush command `drush helfi:remove-non-helsinki-tpr-units`
- see a considerably smaller number of TPR Units in the admin view
